packages/lighthouse/src/index.ts

```diff
@@ -22,11 +22,14 @@ const lhConfig = ({ urls, server, assertions = {} }: Params) => {
       assert: {
         preset: 'lighthouse:no-pwa',
         assertions: {
-          // Final Ligthouse score Budgets
+          // Final Lighthouse score Budgets
           'categories:accessibility': ['error', { minScore: 1 }],
           'categories:best-practices': ['error', { minScore: 1 }],
           'categories:performance': ['error', { minScore: 0.95 }],
-          'categories:seo': ['error', { minScore: 1 }],
+          'categories:seo':
+            process.env.NODE_ENV !== 'production'
```

Comment:
I'm not sure if the preview environment is treated as production or another one.